backend/core/src/main/java/org/sonarsource/sonarlint/core/telemetry/TelemetryServerAttributesProvider.java

```diff
@@ -108,6 +105,12 @@ private int countSonarQubeServerBindings(Collection<BoundScope> allBindings) {
       .count();
   }
 
+  private int countChildBindings(Collection<BoundScope> allBindings) {
+    return (int) allBindings.stream()
+      .filter(scope -> configurationRepository.getParentId(scope.getConfigScopeId()).isPresent())
```

Comment:
Actually, I think this is not sufficient. If you check the implementation of `ConfigurationRepository#getAllBoundScopes`, you will see that all config scopes having an "effective binding" will be returned. And the effective binding is something you can inherit from the parent, so I think you would also count modules not having a override